WeakAuras/Init.lua

```diff
@@ -521,7 +521,38 @@ do
     if C_AddOns.IsAddOnLoaded("!!WWAddOnsFix") then
       noIncompatibleAddon = false
       C_Timer.After(1, function()
-        WeakAuras.prettyPrint("Incompatible addon detected: please disable '!!WWAddOnsFix'")
+        local text = "WWAddOnsFix addon is incompatible with WeakAuras."
+        WeakAuras.prettyPrint(text)
+        StaticPopupDialogs["WA_INCOMPATIBLE_ADDON"] = {
+          text = text,
+          button1 = "Disable WWAddOnsFix",
+          button2 = CANCEL,
+          OnAccept = function()
+              C_AddOns.DisableAddOn("!!WWAddOnsFix")
+              ReloadUI()
+           end,
+          timeout = 0,
+          whileDead = true,
+        }
+        StaticPopup_Show("WA_INCOMPATIBLE_ADDON")
+      end)
+    elseif WeakAuras.IsTWW() and C_CVar.GetCVar("loadDeprecationFallbacks") == "1" then
+      noIncompatibleAddon = false
+      C_Timer.After(1, function()
+        local text = "CVar loadDeprecationFallbacks is enable and is incompatible with WeakAuras."
```

Comment:
"enabled"

What makes loadDeprecationFallbacks incompatible? Are the functions implemented incorrectly?

Reply:
Let me see if i can handle it more gracefully